core/services/keystore/master.go

```diff
@@ -19,6 +19,8 @@ import (
 	"github.com/smartcontractkit/sqlx"
 )
 
+//go:generate mockery --name Master --output ./mocks/ --case=underscore
```

Comment:
looks like this is actually unnecessary (see line 26)

Reply:
Oh wow this must've snuck in with one of the merge commits! Nice catch.

Reply:
Yep. Looks like merge commit 4ab593bea9a7eee2a5daad74b162d1fba2809595 introduced it again.